addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -310,6 +310,7 @@ $o-we-technical-modal-zindex: 2001;
     .modal-content {
         border-radius: 0;
         background-color: white;
+        color: #33363e;
```

Comment:
For `website`'s `o_technical_modal`, restores the `color` value from `o-w-preserve-base` which was overridden by the contrast definition in `web`'s `bootstrap_review.scss`.

Reply:
> What this is doing: For `website`'s `o_technical_modal`, restores the `color` value from `o-w-preserve-base` which was overridden by the contrast definition in `web`'s `bootstrap_review.scss`.

I'll leave it like that, it's quite logical to also have the text color to preserve